index.js

```diff
@@ -1,18 +1,21 @@
 // Q1:
 // Write a JavaScript function that takes a customer's name and account balance and returns a string in the format: "CustomerName has a balance of balance KD."
 function formatCustomerBalance(customerName, balance) {
-  // Write code here
+  const customers = customers.forEach((customer) => balance);
+  return customers;
 }
 
 formatCustomerBalance("John Doe", 150); // => "John Doe has a balance of 150 KD."
 formatCustomerBalance("Jane Doe", 250); // => "John Doe has a balance of 250 KD."
-
+console.log(formatCustomerBalance);
```

Comment:
this is not how we log the return value of the function, you are only logging the function declaration.